### ISSUE TITLE
Unconfirmed transactions not in GUI

### ISSUE THREAD

**Issue by tomasbrod** [gridcoin-community/Gridcoin-Research#1291]
Similar to #1157, the GUI does not show unconfirmed incoming transactions either. I sent myself coins from other wallet. The transaction could be retrieved with `gettransaction`, but it did not show up in gui. It only appeared, accompanied by popup/toast notification just after block update.

**Comment by iFoggz** [comment]
every block it is updated is it not? only shows incoming on 1 conf i believe u got more details

**Comment by tomasbrod** [comment]
Yes, it is updated after a block. But not before.
Incoming transactions that are not in any block are hidden from gui, despite being available to gettransaction (mempool).

**Comment by personthingman2** [comment]
Yes I have also noticed this in the past. However it has not been much of an issue because of our fast block times.

**Comment by jamescowens** [comment]
Do we want to fix then?

**Comment by iFoggz** [comment]
that can be fixed in some way or shape. lets add this to after my PR on side stake, POR/POS detection so the conflict wont be there. idk if the gui side accesses the memory pool

**Comment by jamescowens** [comment]
I agree.

**Comment by sibebleuze** [comment]
I don't know if anyone else is interested in this feature, but I would like to propose that unconfirmed transactions, even when they failed to confirm, would stay in the GUI. Specifically, I would like to be able to see a mined POR block even when it's gone due being on the wrong side of a fork or something. For example, I got a notification from Gridcoin Research about 45 minutes ago that I had staked. I was busy at the time and when I went to check it out, it was gone already. Now like I said, not sure what you guys want, but I would like it if I were able to see it in my GUI with some indication of the failure or something, rather than it being just gone. Would appreciate if you considered this.

**Comment by cyrossignol** [comment]
@sibebleuze Sorry for the late reply. If you add `showorphans=true` to your configuration file, the wallet will display stale (orphaned) stakes as "Generated but not accepted". This option has been available for years, but it was never documented.
